tests/packages/snack.nix

```diff
@@ -1,9 +1,4 @@
-let
-  pkgs = import ../../nix {};
-  snack = pkgs.snack-lib;
-in
-  snack.executable
-    { main = "Foo";
-      src = ./src;
-      dependencies = ["conduit"];
-    }
+{ main = "Foo";
+  src = ./src;
+  dependencies = ["conduit"];
+}
```

Comment:
how do you know if the module is an executable or a library?

Reply:
I need to write some heuristics; basically, if there's no `main` attribute, it's a library. But I also need to add some logic for building libraries...